src/css/content.scss

```diff
@@ -82,7 +82,7 @@
 }
 
 input:checked + .autoplay-slider {
-  background-color: #3f3f3f;
+  background-color: #2dcb45;
```

Comment:
The color doesn't really fit with the default yt shorts interface since the sidebar is just monochrome.

Reply:
I understood your intention.
This change is canceled.